sematic/api/endpoints/organizations.py

```diff
@@ -0,0 +1,24 @@
+# Standard Library
+from typing import Optional
+
+# Third-party
+import flask
+
+# Sematic
+from sematic.api.app import sematic_api
+from sematic.api.endpoints.auth import authenticate
+from sematic.db.models.user import User
+from sematic.db.queries import get_organizations
+
+
+@sematic_api.route("/api/v1/organizations", methods=["GET"])
+@authenticate
+def list_organizations_endpoint(user: Optional[User]) -> flask.Response:
+    """
+    Retrieve the list of organizations.
+    """
+    organizations = get_organizations()
+
+    payload = [organization.to_json_encodable() for organization in organizations]
```

Comment:
Since we are creating new APIs, I recommend we consider these two:

1. JSON conversion (including calling `to_json_encodeable()` and `flask.jsonify()`) can be done in a [after_request](https://flask.palletsprojects.com/en/2.3.x/api/#flask.Flask.after_request) hook. This way we have a dedicated place to convert JSONEncodableMixin universally. Also in the future, we can add logic in reacting to different requested "content-type" (eg. xml) in a centralized way. This API endpoint function only needs to focus on business logic.

2. Together with the above, we should consider adopting [Blueprint](https://tedboy.github.io/flask/generated/generated/flask.Blueprint.html) to divide business domains and scope hooks usage.